metadata/decoration.xml

```diff
@@ -28,12 +28,12 @@
 		<option name="active_color" type="color">
 			<_short>Color when window is active</_short>
 			<_long>Sets the color when the window is active.</_long>
-			<default>#222222aa</default>
+			<default>0.6 0.6 0.6 1.0</default>
```

Comment:
Is there a reason to change these or are these values your preference?

Reply:
I did not know the equivalent in this format.
I took the occasion to experiment variants.